packages/sdk/src/rpc/rpc.ts

```diff
@@ -204,11 +204,15 @@ export class Rpc {
   /**
    * Broadcast a Tx to the ledger
    * @async
-   * @param {SignedTx} signedTx - Transaction with signature
-   * @returns {void}
+   * @param signedTx - Transaction with signature
+   * @returns void
    */
-  async broadcastTx(signedTx: SignedTx): Promise<void> {
+  async broadcastTx(signedTx: SignedTx): Promise<string> {
     const { txMsg, tx } = signedTx;
-    await this.sdk.process_tx(tx, txMsg);
+    return await this.sdk.process_tx(tx, txMsg);
+  }
+
+  async shieldedSync(vks: string[]): Promise<void> {
```

Comment:
Missing docs here.

Reply:
This is weird, jsdoc plugin detects that there is "wrong" description but not that there is none :| I'm pretty sure it worked in the beginning :D